clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1834,6 +1834,14 @@ bool CompilerInvocation::ParseCodeGenArgs(CodeGenOptions &Opts, ArgList &Args,
     Opts.setInlining(CodeGenOptions::NormalInlining);
   }
 
+  // If we have specified -Og and have not set any explicit -fextend-lifetimes
+  // value, then default to -fextend-lifetimes=all.
```

Comment:
This is going to create confusion by speculating the existence of an `={all,params,this}`  fextend-lifetimes when it doesn't exist yet, and might not.

Reply:
Ah, I implemented the `=` option on a local-only branch, and accidentally left this comment in. My mistake, though I may yet decide to merge the feature into the earlier PRs.